cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -20,14 +20,16 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
             set responseJSON = ..GetSettingsURL(%request)
         } elseif $extract(pagePath, 6, *) = "get-package-version"{
             set responseJSON = ..GetPackageVersion()
+        } elseif $extract(pagePath, 6, *) = "namespace"{
+            set responseJSON = $NAMESPACE
```

Comment:
Should be actual JSON for consistency

Reply:
Actually, this seems unused (we just use the URL in the client code) - should just remove.